src/loaders/dts.ts

```diff
@@ -27,6 +27,7 @@ export async function getDeclaration (contents: string, path = '_contents.ts') {
 
     return files[path.replace(/\.(ts|js)$/, '.d.ts')]
   } catch {
+    console.warn(`Could not generate declaration file for ${path}. Do you have \`typescript\` installed?`)
```

Comment:
Exception catching is too board and we are hiding any other potential error that might be happen. We might move `Do you have` part directly after import chain and here include error as second arg